StreemNow/AppDelegate.swift

```diff
@@ -1,50 +1,25 @@
 //
 //  AppDelegate.swift
-//  StreemNow
+//  Streem
 //
-//  Created by Sean Adkinson on 07/17/2018.
 //  Copyright © 2018 Streem, Inc. All rights reserved.
 //
 
 import UIKit
+import Streem
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
-
-    func applicationWillResignActive(_ application: UIApplication) {
-        NSLog(#function)
-        // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
-        // Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
-    }
-
-    func applicationDidEnterBackground(_ application: UIApplication) {
-        NSLog(#function)
-        // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
-        // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
-    }
-
-    func applicationWillEnterForeground(_ application: UIApplication) {
-        NSLog(#function)
-        // Called as part of the transition from the background to the inactive state; here you can undo many of the changes made on entering the background.
-    }
-
-    func applicationDidBecomeActive(_ application: UIApplication) {
-        NSLog(#function)
-        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
-    }
-
-    func applicationWillTerminate(_ application: UIApplication) {
-        NSLog(#function)
-        // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+    
+    func application(_ application: UIApplication, handleEventsForBackgroundURLSession identifier: String, completionHandler: @escaping () -> Void) {
+        Streem.setBackgroundURLSession(withIdentifier: identifier, completionHandler: completionHandler)
```

Comment:
Would all clients need to add this line to their `AppDelegate`?  If so, let's add to the README

Reply:
Good catch!